### DISCUSSION TITLE
Welcome to Product_3D Discussions!

### DISCUSSION THREAD

**Discussion by amanjaiman1** [amanjaiman1/Product_3D#34] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by amanjaiman1** [comment]
Hey guys , how r u all and please come openly to discuss each other what are you doing and how's your project contribution going on

**Comment by balaji-sivasakthi** [comment]
Hi @amanjaiman1, I am Balaji Sivasakthi from Coimbatore, Tamil Nadu, India. I am a full-stack developer with around 2 years of experience. I am happy to be here!

**Comment by amanjaiman1** [comment]
Me too bro

**Reply by balaji-sivasakthi** [reply]
That's great!
